### ISSUE TITLE
Scheduled task for running the app at startup results in error

### ISSUE THREAD

**Issue by balintlerch** [kulvind3r/GamingGaiden#5]
### Problem

The scheduled task on Windows startup results in the following error and does not start the application. Double clicking the shortcut still runs the app.

### Error Image

![image](https://github.com/kulvind3r/GamingGaiden/assets/33422573/37c53ee4-297a-4aea-a39e-7974f2fad148)

**Comment by kulvind3r** [comment]
Not sure what is happening with your system. Scheduled Task does exactly the same thing that the shortcut does. No difference.

The above error is related to HWinfo sensors. Basically application is trying to set values of the sensor at startup.

One possibility that could be happening is that application is starting very quickly before Operating system even allows any app to access registry. So at startup registry access is still not ready and when app tries to write to registry it gets this failure.

and when later you run it from shortcut , registry is ready by then to be accessed by application and it works

You can try editing the scheduled task to start after a delay of 1 min and see if that fixes the error.

**Comment by balintlerch** [comment]
Unfortunately, adding a delay didn't work. I don't really know what causes the issue. If I find a solution, I'll update here.

**Comment by kulvind3r** [comment]
Few questions to debug...

1. Do you have  HWinfo installed? If you have the gaming gaiden sensors will appear in hwinfo and if they are appearing that means the path exist and the application is working as expecting.
2. Can you check if the path does exist in the registry?

**Comment by balintlerch** [comment]
So, I had a faulty HWinfo installation. After reinstalling it the error disappeared. 
However, the scheduled task still could not start. I still had the PowerShell popup on each Windows startup saying that Thread Job's state is Not Started (so basically the picture I attached without all the red stuff 😂). 

I found a workaround, however. I just disabled the scheduled task and put the Gaming Gaiden shortcut in my PC's startup folder. Now the app is successfully launched with all my other startup applications. So, this issue is solved on my end.

Thank You for your quick support though.
